hangups/ui/__main__.py

```diff
@@ -836,6 +838,40 @@ def main():
         with open(default_config_path, 'a') as cfg:
             cfg.write("")
 
+    # Create empty colors directory if there is none
+    dir_maker(default_colors_path)
+
+    # Read .col files
+    attributes = {'active_tab', 'inactive_tab', 'msg_date', 'msg_sender', 'msg_text', 'status_line', 'tab_background'}
+    for root, dirs, files in os.walk(default_colors_path):
+        for filename in files:
+            splitName = os.path.splitext(filename)
+            if splitName[1] == '.json':
+                obj = []
+                usedAttrs = []
+
+                name = splitName[0]
+
+                with open(os.path.join(root, filename), "r") as f:
+                    try:
+                        data = json.load(f)
+                    except:
+                        print("Error loading JSON file (" + filename + ")")
+                        return
```

Comment:
It's always better to catch a specific exceptions, and you can use `sys.exit` to both exit with status 1 and print the message to stderr:
```python
except ValueError:
    sys.exit('Error loading JSON file: {}'.format(filename))
```